### DISCUSSION TITLE
Add Disk space utilization Pie to System Overwiew

### DISCUSSION THREAD

**Discussion by fabriba** [netdata/netdata#13478] [Q&A]
Hi there,
probably a newby question..

I'd like to add just at the top of my overview, an indicator of how much disk space I am using for a given disk (sda in my case).
Ideally in the same style as the "Used Ram" in the default.

I saw there are ways to create your own dashboard but it seems like this would likely have a simpler solution than building the dashboard from scratch and having to maintain it.
Does anyone know how to do this?

**Comment by vlvkobal** [comment]
There are no simpler solutions on the agent side. I examined a bit `main.js` and it looks like there are no methods for in-place configuration for System Overview gauges (@jacekkolasa correct me if I'm wrong).

Anyway, the better way to customize your dashboard is to use [Netdata Cloud](https://learn.netdata.cloud/docs/cloud). If you'd like to make any feature request for the UI, Cloud has a much higher priority than the agent.

**Comment by thiagoftsm** [comment]
Hello @fabriba ,

As @vlvkobal said probably there is not a simple way, but you can try to work with [badges](https://learn.netdata.cloud/docs/agent/web/api/badges) and [custom dashboards](https://learn.netdata.cloud/docs/agent/web/gui/custom).

Best regards!

**Comment by jacekkolasa** [comment]
@vlvkobal @fabriba The only way to customise it in Agent Dashboard would be to create custom dashboard_info.js (as you said, it's hard to build dashboard from scratch). You can customize heads and mainheads, as described [here](https://github.com/netdata/netdata/blob/master/web/gui/dashboard_info.js#L1082-L1093) , [example](https://github.com/netdata/netdata/blob/master/web/gui/dashboard_info.js#L1680).

You can learn about custom dashboard_info js here: https://learn.netdata.cloud/docs/dashboard/customize

But overall, building your own dashboard is easiest in Netdata Cloud
